src/locales/en.json

```diff
@@ -190,7 +190,7 @@
     "1person_15minCarRide": "Car ride with 1 person for 15 mins",
     "oneNightStand": "One-night stand with a random person",
     "liveInPartner_noContacts": "Live-in partner who has no indoor interactions besides you",
```

Comment:
So the live-in partner can have other outdoor interactions? A bit strange this would be a category…

Reply:
What do you find strange about that? It's very common in our experience for a "closed bubble" to allow seeing friends outdoors, masked, distanced, without that being a violation of their bubble agreement.

Reply:
Bubble agreement? How is grocery shopping not an indoor interaction? Does it mean personal interactions or somesuch? Since there is no timespan here (further questioning the distinction), it seems almost nobody would qualify.

Reply:
I agree that the risk categories have really poor wording. I don't know how to put enough information into them to specify what they mean without overloading the user.

Reply:
@beshaya Isn't it either "Close contact", (which seems to be the common nomenclature), or "intimate partner"?
@catherio What is the "bubble agreement", and what is it meant to do?